### PR TITLE
Clarify the heading for instructions to migrate a host

### LINKED ISSUE #62: Document how to move a host to a new env
From a customer:

> Is there some documentation/instruction on how to properly migrate a host to a different environment for VC?

Reply:

shut down the agents, edit /etc/vividcortex/global.conf and put the correct api token in there, then restart. You can find the right token by opening up the desired environment, opening the Add Host wizard and looking there. The metrics will remain, but in the old environment.

### PR DESCRIPTION
**Pull request by potocnyj**
Specify that these instructions are for moving a host to a new
environment to avoid ambiguity. Additionally add a sentence noting
where a user can find the API token when moving a host.

Closes #62

### CODE REVIEW

**Comment by potocnyj** [comment]
@Preetam PTAL

File: content/getting-started/managing-hosts.md [hunk 1]
```diff
@@ -39,11 +39,11 @@ Once a host is deleted it's also a good idea to disable the service (for example
 `chkconfig --del vividcortex`) so it doesn't restart. You can also
 [uninstall](/getting-started/installing/ "Documentation about the installation") the agents permanently.
 
-## Moving a Host
+## Moving a Host to a different Environment
```
Comment (Preetam) [inline content/getting-started/managing-hosts.md]: Why is "environment" capitalized?

File: content/getting-started/managing-hosts.md [hunk 2]
```diff
@@ -39,11 +39,11 @@ Once a host is deleted it's also a good idea to disable the service (for example
...
 
 You can move a host to a new environment simply by stopping the agents, changing
 the `/etc/vividcortex/global.conf` file to contain the new environment's API token,
-and restarting the agents. However, it is *not* possible to move a host's data to
+and restarting the agents. The API token can be found in the "Add Host" Wizard while in the destination environment. Note that it is *not* possible to move a host's data to
```
Comment (Preetam) [inline content/getting-started/managing-hosts.md]: Please keep the line widths consistent.

```
## Moving a Host to a different Environment

You can move a host to a new environment simply by stopping the agents, changing
the `/etc/vividcortex/global.conf` file to contain the new environment's API token,
and restarting the agents. The API token can be found in the "Add Host" Wizard while in the destination environment. Note that it is *not* possible to move a host's data to
a new environment at this time. Environments are isolated from each other by design,
and data is not portable across environments.
```

File: content/getting-started/managing-hosts.md (hunk 1 above)
Comment (potocnyj) [inline content/getting-started/managing-hosts.md]: *shrug*, I'll lowercase it

File: content/getting-started/managing-hosts.md (hunk 2 above)
Comment (potocnyj) [inline content/getting-started/managing-hosts.md]: Done

**Pull request merged**